src/Presentation/Console/Commands/UpdateCommand.php

```diff
@@ -125,7 +125,7 @@ protected function checkForUpdate(OutputInterface $output)
 
         $latestVersion = $this->getLatestVersion($output);
 
-        if (version_compare($currentVersion, $latestVersion) === -1) {
+        if (version_compare($currentVersion, $latestVersion, '>')) {
```

Comment:
Shouldn't there be `<` sign so the condition would be `if ($currentVersion < $latestVersion)` ?

Reply:
the latest version is what we had, the current version is what we have now.